deploy/scripts/src/ipv-core/test.ts

```diff
@@ -209,9 +207,11 @@ export function passport (): void {
     {
       'is status 302': (r) => r.status === 302
     })
-    res = timeRequest(() => http.get(res.headers.Location, {
-      tags: { name: 'B01_Passport_05_ContOnPYIStartPage_02_PassStub' }
-    }),
+    res = timeRequest(() => {
+      const response = http.get(res.headers.Location, { tags: { name: 'B01_Passport_05_ContOnPYIStartPage_02_PassStub' } })
+      if (env.staticResources) getStaticResources(response)
+      return response
+    },
```

Comment:
```suggestion
    res = timeRequest(() => http.get(res.headers.Location, { tags: { name: 'B01_Passport_05_ContOnPYIStartPage_02_PassStub' } }),
```
Don't retrieve static resources on the stub call